src/elements/content-sidebar/Sidebar.js

```diff
@@ -127,6 +127,13 @@ class Sidebar extends React.Component<Props, State> {
             this.setState({ isDirty: true });
         }
 
+        if (boxAISidebarProps && boxAISidebarProps.shouldOpenBoxAISidebar) {
+            // navigate to boxai tab and open sidebar
+            history.push(`/${SIDEBAR_VIEW_BOXAI}`);
```

Comment:
Is this needed given the redirect logic already present in [SidebarPanels](https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/SidebarPanels.js#L372)? It should open by default. It would be great to avoid adding complexity to the routing, if possible.

Reply:
It's needed for current Box AI button that is now opening modal. So we need a way to open this sidebar from a different control

Reply:
That's already possible via the use of the `sb` query parameter and the default routing rules, though.

Reply:
Oh, you're right, I didn't think about that! In that case these changes are not needed 👍